README.md

```diff
@@ -84,6 +84,21 @@ two properties in the "Common" section
 copy of the `./config/marklogic-connect-standalone.properties` file. This file already has the two properties set to the
   correct values, so you should not need to do anything further with them.
 
+### Dead Letter Configuration
+
+Starting with version 1.8.0, the sink connector makes use of the dead letter queue (DLQ) if the user has configured
+Kafka to use the DLQ. For information about configuring Kafka to use the DLQ, please refer to:
+
+    https://www.confluent.io/blog/kafka-connect-deep-dive-error-handling-dead-letter-queues/
```

Comment:
Putting this in a code block makes it unclickable - see:

![image](https://user-images.githubusercontent.com/5026193/211880720-933b6f41-f0b1-4437-9db4-a3cce65943f9.png)

I recommend doing this instead (i.e. helpful text in brackets, followed by link in parens):

[Please see the Kafka documentation](https://www.confluent.io/blog/kafka-connect-deep-dive-error-handling-dead-letter-queues/) for more information on configuring the dead letter queue in Kafka.